sections/ansible.zsh

```diff
@@ -0,0 +1,50 @@
+#
+# Ansible
+#
+# Ansible is an IT automation tool. It can configure systems, deploy software, and orchestrate more advanced IT tasks
+# such as continuous deployments or zero downtime rolling updates.
+# Link: http://docs.ansible.com/ansible/latest/index.html
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_ANSIBLE_SHOW="${SPACESHIP_ANSIBLE_SHOW=true}"
+SPACESHIP_ANSIBLE_SHOW_VERSION="${SPACESHIP_ANSIBLE_SHOW_VERSION=false}"
+SPACESHIP_ANSIBLE_PREFIX="${SPACESHIP_ANSIBLE_PREFIX="$SPACESHIP_PROMPT_DEFAULT_PREFIX"}"
+SPACESHIP_ANSIBLE_SUFFIX="${SPACESHIP_ANSIBLE_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_ANSIBLE_SYMBOL="${SPACESHIP_ANSIBLE_SYMBOL="🅐"}"
+SPACESHIP_ANSIBLE_COLOR="${SPACESHIP_ANSIBLE_COLOR="white"}"
+
+# ------------------------------------------------------------------------------
+# Section
+# ------------------------------------------------------------------------------
+
+# Show ansible status
+# spaceship_ prefix before section's name is required!
+# Otherwise this section won't be loaded.
+spaceship_ansible() {
+  # If SPACESHIP_ANSIBLE_SHOW is false, don't show ansible section
+  [[ ${SPACESHIP_ANSIBLE_SHOW} == false ]] && return
+
+  # Check if ansible command is available for execution
+  spaceship::exists ansible || return
```

Comment:
Why not just disable version display instead?

Reply:
Version display is disabled by default.

Reply:
oh, okay. so I'd just drop this check

Reply:
dropping the check